cypress/integration/common/common.js

```diff
@@ -0,0 +1,27 @@
+import { Given, Then } from 'cypress-cucumber-preprocessor/steps'
+import { dashboards } from '../../assets/backends'
+
+beforeEach(() => {
+    cy.visit('/', {
+        timeout: 15000,
+    })
+})
+
+Given('I open the {string} dashboard', title => {
+    cy.get('[data-test="dhis2-uicore-chip"]', {
```

Comment:
Can this be targeted more specifically by adding a custom data-test prop? `dhis2-uicore-chip` could be used in other places?

Reply:
`view-dashboard-title`?

Reply:
Done. Using 'dashboard-chip'